Source/Scene/Cesium3DTileset.js

```diff
@@ -786,6 +802,68 @@ define([
 
     ///////////////////////////////////////////////////////////////////////////
 
+    var scratchJulianDate = new JulianDate();
+
+    function updateExpireDate(tile) {
+        if (defined(tile.expireDuration)) {
```

Comment:
Are you sure this code is right?  What is duration is defined, but not date?  Looks like it is ignored, right?

I suggest that the spec should say say to take the earlier of the two, but we could also just disallow defining both if we think it doesn't rule out any important use cases.